nomenclature/region_mapping_models.py

```diff
@@ -115,7 +154,30 @@ def check_illegal_renaming(cls, values):
         return values
 
     @classmethod
-    def create_from_region_mapping(cls, file: Union[Path, str]):
+    def from_file(cls, file: Union[Path, str]):
+        """Effectively the constructor for RegionAggregationMapping.
+
+        Parameters
+        ----------
+        file : Union[Path, str]
```

Comment:
The layout is different to other modules e.g. _core.py_.

Reply:
What do you mean by that?

Reply:
Ah sorry, all good, I checked wrong.